src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -405,22 +405,46 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
         }
     }
 
+    /**
+     * Emit an EncryptionKeyChanged event for every tracked encryption keys.
+     */
+    public emitEncryptionKeys(): void {
```

Comment:
How are external callers meant to use this?  The one obvious usage is in [Element Call's setup of RTC session keys](https://github.com/element-hq/element-call/blob/ed35d6b3778adff519cb0e9778ae2c31e47bda6a/src/e2ee/matrixKeyProvider.ts#L39).  Based on that, is the intention for this to be used for startup logic / to behave as though all keys have changed?

Reply:
Yes, I've attempted to articulate this in the inline documentation.